### PR TITLE
Nextcloud documentation changes

### PR DESCRIPTION
**Pull request by MayaBerd**

### CODE REVIEW

File: docs/user-guide/nextcloud-integration/README.md [hunk 1]
```diff
@@ -174,6 +174,18 @@ There are two additional features related to the integration that you can enable
 
 ![Nextcloud search for work packages](nc_global_search.png)
 
+## Work package link preview in Nextcloud
+
+Starting with OpenProject 13.0 a work package link preview will be shown if you use Nextcloud Talk or Text apps. You can [copy a work package link](../work-packages/copy-move-delete/#copy-link-to-clipboard) and paste it into a Talk or Text app. Alternatively you can use **/** to activate the **smart picker** and find the work package by searching.
```
Comment (wielinde) [inline docs/user-guide/nextcloud-integration/README.md:179]: @MayaBerd Actually the link preview is a feature purely on the Nextcloud side. So the relevant version is the one of the Nextcloud app (2.4) and not OpenProject

**Comment by wielinde** [comment]
@MayaBerd The content of this PR looks all very good. I am happy that you already merged it. Please find my little note on a tiny detail above.

**Comment by MayaBerd** [comment]
> @MayaBerd The content of this PR looks all very good. I am happy that you already merged it. Please find my little note on a tiny detail above.

@wielinde thank you, I will update it in a separate PR!

File: docs/user-guide/nextcloud-integration/README.md (hunk 1 above)
Comment (MayaBerd) [reply docs/user-guide/nextcloud-integration/README.md:179]: making the change here: https://github.com/opf/openproject/pull/13528

**Pull request merged**